doc/command_ref.rst

```diff
@@ -1449,6 +1476,17 @@ Updateinfo Command
 
     Output of the ``--summary`` option is affected by the :ref:`autocheck_running_kernel <autocheck_running_kernel-label>` configuration option.
 
+
+.. _update_minimal_command-label:
+
+----------------------
+Update-minimal Command
+----------------------
+
+``dnf [options] update-minimal``
+    Deprecated alias for the :ref:`\upgrade-minimal_command-label`.
```

Comment:
The `upgrade-minimal_command-label` doesn't exist, it's `upgrade_minimal_command-label`. Maybe the one with the dash is the correct one though, as the command is called `upgrade-minimal`? It looks inconsistent as most of the commands which have a dash in the name have an underscore instead of the dash in the label, but not all of them. It probably doesn't matter much which way, but could you maybe unify it while at it? I'd go with the dash myself.

Reply:
I added commit that unifies the labels and also headings. And I went with underscore, because it's much less changes and I don't have to differentiate between such commands and for example "module enable" or "history list".

I will just check if there are no references from dnf-plugins-core or something.

Reply:
Ok. Seems to me there aren't any references from dnf-plugins-{core,extras}. Let me know when you're done checking and I'll merge this. Thanks!

Reply:
Same for me. Thanks for the review!